registry/storage/driver/s3-aws/s3.go

```diff
@@ -1167,15 +1167,24 @@ func (d *driver) doWalk(parentCtx context.Context, objectCount *int64, path, pre
 // the previous and current paths in sorted order.
 //
 // Eg 1 directoryDiff("/path/to/folder", "/path/to/folder/folder/file")
-//   => [ "/path/to/folder/folder" ],
+//
+//	=> [ "/path/to/folder/folder" ],
```

Comment:
Ah! hehe. this will conflict with my `gofumpt` PR (https://github.com/distribution/distribution/pull/3766)